source/reference/best-practices.rst

```diff
@@ -113,4 +113,4 @@ libstdc++.so.6
 --------------
 
 .. note::
-	**As a general rule of thumb, please use no libstdc++.so.6 newer than the one that comes with the oldest distribution that you still want to support, i.e., the oldest still-supported LTS version** (at the time of this writing, Ubuntu 14.04).
+	**As a general rule of thumb, please use no libstdc++.so.6 newer than the one that comes with the oldest distribution that you still want to support, i.e., the oldest still-supported LTS version** (at the time of this writing, Ubuntu 16.04).
```

Comment:
Maintaining any sort of version number is a bad idea. Here, combined with the lack of a date when this was last updated ("at the time of writing", well, when was that?), the reader is confused. See above.

Two options: add a date when this was last updated, or, better, do not put in any hardcoded version number but place a link to up to date information.